.storybook/webpack.config.js

```diff
@@ -39,7 +39,15 @@ module.exports = {
                     path.join(__dirname, '../stories'),
                     path.join(__dirname, '../src')
                 ],
-                loaders: ['awesome-typescript-loader?configFileName=./stories/tsconfig.json']
```

Comment:
Why did you migrate to `ts-loader`? At least in my experience `awesome-typescript-loader` is usually quite faster.
We use `awesome-typescript-loader` in KD (storybooks) and it works just fine :)

Reply:
Yeah, i tested ts-loader with `transpileOnly: true` and it's same speed. And i think there are no other benefits using 3rd party loader.